ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/valueobject/ViewDeletedEvent.java

```diff
@@ -0,0 +1,15 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.view.valueobject;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.viewcreation.valueobjects.ViewName;
+
+public class ViewDeletedEvent {
```

Comment:
This COULD be a record. But i presume you have a reason why you prefer to not have it like that?